src/main/jte/admin/confirmation/disable_guests.jte

```diff
@@ -11,6 +11,6 @@
 <div class="modal-action">
     @template.util.modal.close(message = context.localize("generic.cancel"), id = id)
     @template.util.form(csrf = csrf, cssClass = "card-actions justify-end", action = "/admin/users/disable-guests", method = PUT, content = @`
-        <input class="btn btn-error" type="submit" value="${context.localize("administration.user.guests.disable")}">
+        <input class="btn btn-error" type="submit" value="${context.localize("administration.user.guests.disable")}" aria-label="${context.localize("administration.user.guests.disable")}">
```

Comment:
This text is already the text of the button, I don't think repeating it is necessary.